Test failures with PROJ 9.7.1

The Debian package build fails with PROJ 9.7.1 due to test failures:
```python
=================================== FAILURES ===================================
____________________________ test_datum_horizontal _____________________________

    def test_datum_horizontal():
>       assert CRS.from_epsg(5972).datum == CRS.from_epsg(25832).datum
E       assert DATUM["ETRS89-NOR [EUREF89]",\n    ELLIPSOID["GRS 1980",6378137,298.257222101,\n        LENGTHUNIT["metre",1]],\n    ANCHOREPOCH[1995],\n    ID["EPSG",1407]] == ENSEMBLE["European Terrestrial Reference System 1989 ensemble",\n    MEMBER["European Terrestrial Reference Frame 1989",\n        ID["EPSG",1178]],\n    MEMBER["European Terrestrial Reference Frame 1990",\n        ID["EPSG",1179]],\n    MEMBER["European Terrestrial Reference Frame 1991",\n        ID["EPSG",1180]],\n    MEMBER["European Terrestrial Reference Frame 1992",\n        ID["EPSG",1181]],\n    MEMBER["European Terrestrial Reference Frame 1993",\n        ID["EPSG",1182]],\n    MEMBER["European Terrestrial Reference Frame 1994",\n        ID["EPSG",1183]],\n    MEMBER["European Terrestrial Reference Frame 1996",\n        ID["EPSG",1184]],\n    MEMBER["European Terrestrial Reference Frame 1997",\n        ID["EPSG",1185]],\n    MEMBER["European Terrestrial Reference Frame 2000",\n        ID["EPSG",1186]],\n    MEMBER["European Terrestrial Reference Frame 2005",\n        ID["EPSG",1204]],\n    MEMBER["European Terrestrial Reference Frame 2014",\n        ID["EPSG",1206]],\n    MEMBER["European Terrestrial Reference Frame 2020",\n        ID["EPSG",1382]],\n    ELLIPSOID["GRS 1980",6378137,298.257222101,\n        LENGTHUNIT["metre",1],\n        ID["EPSG",7019]],\n    ENSEMBLEACCURACY[0.1],\n    ID["EPSG",6258]]
E        +  where DATUM["ETRS89-NOR [EUREF89]",\n    ELLIPSOID["GRS 1980",6378137,298.257222101,\n        LENGTHUNIT["metre",1]],\n    ANCHOREPOCH[1995],\n    ID["EPSG",1407]] = <Compound CRS: EPSG:5972>\nName: ETRS89-NOR [EUREF89] / UTM zone 32N + NN2000 height\nAxis Info [cartesian|vertical]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\n- H[up]: Gravity-related height (metre)\nArea of Use:\n- name: Norway - onshore - between 6°E and 12°E.\n- bounds: (6.0, 57.9, 12.01, 67.58)\nDatum: ETRS89-NOR [EUREF89]\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\nSub CRS:\n- ETRS89-NOR [EUREF89] / UTM zone 32N\n- NN2000 height\n.datum
E        +    where <Compound CRS: EPSG:5972>\nName: ETRS89-NOR [EUREF89] / UTM zone 32N + NN2000 height\nAxis Info [cartesian|vertical]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\n- H[up]: Gravity-related height (metre)\nArea of Use:\n- name: Norway - onshore - between 6°E and 12°E.\n- bounds: (6.0, 57.9, 12.01, 67.58)\nDatum: ETRS89-NOR [EUREF89]\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\nSub CRS:\n- ETRS89-NOR [EUREF89] / UTM zone 32N\n- NN2000 height\n = from_epsg(5972)
E        +      where from_epsg = CRS.from_epsg
E        +  and   ENSEMBLE["European Terrestrial Reference System 1989 ensemble",\n    MEMBER["European Terrestrial Reference Frame 1989",\n        ID["EPSG",1178]],\n    MEMBER["European Terrestrial Reference Frame 1990",\n        ID["EPSG",1179]],\n    MEMBER["European Terrestrial Reference Frame 1991",\n        ID["EPSG",1180]],\n    MEMBER["European Terrestrial Reference Frame 1992",\n        ID["EPSG",1181]],\n    MEMBER["European Terrestrial Reference Frame 1993",\n        ID["EPSG",1182]],\n    MEMBER["European Terrestrial Reference Frame 1994",\n        ID["EPSG",1183]],\n    MEMBER["European Terrestrial Reference Frame 1996",\n        ID["EPSG",1184]],\n    MEMBER["European Terrestrial Reference Frame 1997",\n        ID["EPSG",1185]],\n    MEMBER["European Terrestrial Reference Frame 2000",\n        ID["EPSG",1186]],\n    MEMBER["European Terrestrial Reference Frame 2005",\n        ID["EPSG",1204]],\n    MEMBER["European Terrestrial Reference Frame 2014",\n        ID["EPSG",1206]],\n    MEMBER["European Terrestrial Reference Frame 2020",\n        ID["EPSG",1382]],\n    ELLIPSOID["GRS 1980",6378137,298.257222101,\n        LENGTHUNIT["metre",1],\n        ID["EPSG",7019]],\n    ENSEMBLEACCURACY[0.1],\n    ID["EPSG",6258]] = <Projected CRS: EPSG:25832>\nName: ETRS89 / UTM zone 32N\nAxis Info [cartesian]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\nArea of Use:\n- name: Europe between 6°E and 12°E: Austria; Denmark - onshore and offshore; Germany - onshore and offshore; Italy - onshore and offshore; Norway including Svalbard - onshore and offshore; Spain - offshore.\n- bounds: (6.0, 36.53, 12.01, 84.01)\nCoordinate Operation:\n- name: UTM zone 32N\n- method: Transverse Mercator\nDatum: European Terrestrial Reference System 1989 ensemble\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\n.datum
E        +    where <Projected CRS: EPSG:25832>\nName: ETRS89 / UTM zone 32N\nAxis Info [cartesian]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\nArea of Use:\n- name: Europe between 6°E and 12°E: Austria; Denmark - onshore and offshore; Germany - onshore and offshore; Italy - onshore and offshore; Norway including Svalbard - onshore and offshore; Spain - offshore.\n- bounds: (6.0, 36.53, 12.01, 84.01)\nCoordinate Operation:\n- name: UTM zone 32N\n- method: Transverse Mercator\nDatum: European Terrestrial Reference System 1989 ensemble\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\n = from_epsg(25832)
E        +      where from_epsg = CRS.from_epsg

test/crs/test_crs.py:420: AssertionError
_________________________________ test_sub_crs _________________________________

    def test_sub_crs():
        crs = CRS.from_epsg(5972)
        sub_crs_list = crs.sub_crs_list
        assert len(sub_crs_list) == 2
>       assert sub_crs_list[0] == CRS.from_epsg(25832)
E       assert <Projected CRS: EPSG:11022>\nName: ETRS89-NOR [EUREF89] / UTM zone 32N\nAxis Info [cartesian]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\nArea of Use:\n- name: Norway including Svalbard south of 84°N between 6°E and 12°E, onshore and offshore.\n- bounds: (5.99, 56.88, 12.0, 84.0)\nCoordinate Operation:\n- name: UTM zone 32N\n- method: Transverse Mercator\nDatum: ETRS89-NOR [EUREF89]\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\n == <Projected CRS: EPSG:25832>\nName: ETRS89 / UTM zone 32N\nAxis Info [cartesian]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\nArea of Use:\n- name: Europe between 6°E and 12°E: Austria; Denmark - onshore and offshore; Germany - onshore and offshore; Italy - onshore and offshore; Norway including Svalbard - onshore and offshore; Spain - offshore.\n- bounds: (6.0, 36.53, 12.01, 84.01)\nCoordinate Operation:\n- name: UTM zone 32N\n- method: Transverse Mercator\nDatum: European Terrestrial Reference System 1989 ensemble\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\n
E        +  where <Projected CRS: EPSG:25832>\nName: ETRS89 / UTM zone 32N\nAxis Info [cartesian]:\n- E[east]: Easting (metre)\n- N[north]: Northing (metre)\nArea of Use:\n- name: Europe between 6°E and 12°E: Austria; Denmark - onshore and offshore; Germany - onshore and offshore; Italy - onshore and offshore; Norway including Svalbard - onshore and offshore; Spain - offshore.\n- bounds: (6.0, 36.53, 12.01, 84.01)\nCoordinate Operation:\n- name: UTM zone 32N\n- method: Transverse Mercator\nDatum: European Terrestrial Reference System 1989 ensemble\n- Ellipsoid: GRS 1980\n- Prime Meridian: Greenwich\n = from_epsg(25832)
E        +    where from_epsg = CRS.from_epsg

test/crs/test_crs.py:574: AssertionError
=============================== warnings summary ===============================
test/test_datadir.py::test_get_data_dir__from_user[str]
test/test_datadir.py::test_get_data_dir__from_user[Path]
test/test_datadir.py::test_append_data_dir__internal[str]
test/test_datadir.py::test_append_data_dir__internal[Path]
  /usr/lib/python3/dist-packages/pyproj/datadir.py:38: UserWarning: pyproj unable to set PROJ database path.
    _set_context_data_dir()

test/test_doctest_wrapper.py::test_doctests
  /usr/lib/python3/dist-packages/pyproj/geod.py:823: UserWarning: Back azimuth is being returned by default to be compatible with inv()This is a breaking change for pyproj 3.5+.To avoid this warning, set return_back_azimuth=True.Otherwise, to restore old behaviour, set return_back_azimuth=False.This warning will be removed in future version.
    warnings.warn(

test/test_doctest_wrapper.py::test_doctests
  /usr/lib/python3/dist-packages/pyproj/geod.py:675: UserWarning: Back azimuth is being returned by default to be compatible with fwd()This is a breaking change for pyproj 3.5+.To avoid this warning, set return_back_azimuth=True.Otherwise, to restore old behaviour, set return_back_azimuth=False.This warning will be removed in future version.
    warnings.warn(

test/test_geod.py: 72 warnings
  /usr/lib/python3/dist-packages/pyproj/geod.py:398: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
    return self._inv_point(

test/test_geod.py: 20 warnings
  /usr/lib/python3/dist-packages/pyproj/geod.py:306: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
    return self._fwd_point(

test/test_transform.py::test_transform_single_point_nad83_to_nad27
  /tmp/autopkgtest-lxc.rhjhs805/downtmp/autopkgtest_tmp/build/test/test_transform.py:64: FutureWarning: This function is deprecated. See: https://pyproj4.github.io/pyproj/stable/gotchas.html#upgrading-to-pyproj-2-from-pyproj-1
    x2, y2 = transform(p1, p2, x1, y1)

test/test_transform.py::test_transform_tuple_nad83_to_nad27
  /tmp/autopkgtest-lxc.rhjhs805/downtmp/autopkgtest_tmp/build/test/test_transform.py:107: FutureWarning: This function is deprecated. See: https://pyproj4.github.io/pyproj/stable/gotchas.html#upgrading-to-pyproj-2-from-pyproj-1
    x2, y2 = transform(p1, p2, x1, y1)

test/test_transformer.py: 40 warnings
  /usr/lib/python3/dist-packages/pyproj/transformer.py:816: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
    return self._transformer._transform_point(

test/test_transformer.py::test_transformer_group_allow_superseded_filter
  /usr/lib/python3/dist-packages/pyproj/transformer.py:207: UserWarning: Best transformation is not available due to missing Grid(short_name=au_icsm_National_84_02_07_01.tif, full_name=, package_name=, url=https://cdn.proj.org/au_icsm_National_84_02_07_01.tif, direct_download=True, open_license=True, available=False)
    super().__init__(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED test/crs/test_crs.py::test_datum_horizontal - assert DATUM["ETRS89-NOR...
FAILED test/crs/test_crs.py::test_sub_crs - assert <Projected CRS: EPSG:11022...
= 2 failed, 941 passed, 3 skipped, 58 deselected, 2 xpassed, 141 warnings in 24.80s =
```

See also: https://github.com/OSGeo/PROJ/blob/master/NEWS.md#971

--- a/test/conftest.py
+++ b/test/conftest.py
@@ -14,7 +14,7 @@ _NETWORK_ENABLED = pyproj.network.is_network_enabled()
 PROJ_LOOSE_VERSION = version.parse(pyproj.__proj_version__)
 PROJ_GTE_941 = PROJ_LOOSE_VERSION >= version.parse("9.4.1")
 PROJ_GTE_95 = PROJ_LOOSE_VERSION >= version.parse("9.5.0")
-PROJ_GTE_97 = PROJ_LOOSE_VERSION >= version.parse("9.7.0")
+PROJ_GTE_971 = PROJ_LOOSE_VERSION >= version.parse("9.7.1")
 
 
 def unset_data_dir():

--- a/test/crs/test_crs.py
+++ b/test/crs/test_crs.py
@@ -20,7 +20,7 @@ from pyproj.crs.enums import CoordinateOperationType, DatumType
 from pyproj.enums import ProjVersion, WktVersion
 from pyproj.exceptions import CRSError
 from pyproj.transformer import TransformerGroup
-from test.conftest import PROJ_GTE_97, PROJ_GTE_941, assert_can_pickle, grids_available
+from test.conftest import PROJ_GTE_941, PROJ_GTE_971, assert_can_pickle, grids_available
 
 
 class CustomCRS:
@@ -418,11 +418,11 @@ def test_datum():
 
 def test_datum_horizontal():
     # EPSG:5972 is a Norwegian compound CRS
-    # In PROJ < 9.7.0, it uses the generic ETRS89 ensemble (same as EPSG:25832)
-    # In PROJ >= 9.7.0 (EPSG v12.022+), it uses the
+    # In PROJ < 9.7.1, it uses the generic ETRS89 ensemble (same as EPSG:25832)
+    # In PROJ >= 9.7.1 (EPSG v12.022+), it uses the
     # Norway-specific ETRS89-NOR realization
     crs_5972 = CRS.from_epsg(5972)
-    if PROJ_GTE_97:
+    if PROJ_GTE_971:
         # Test against ETRS89-NOR [EUREF89] / UTM zone 32N
         crs_11022 = CRS.from_epsg(11022)
         assert crs_5972.datum is not None
@@ -585,9 +585,9 @@ def test_sub_crs():
     sub_crs_list = crs.sub_crs_list
     assert len(sub_crs_list) == 2
     # First sub-CRS should be a projected CRS (UTM zone 32N)
-    # In PROJ < 9.7.0, it's EPSG:25832 (ETRS89 / UTM zone 32N)
-    # In PROJ >= 9.7.0 (EPSG v12.022+), it's EPSG:11022 (ETRS89-NOR / UTM zone 32N)
-    if PROJ_GTE_97:
+    # In PROJ < 9.7.1, it's EPSG:25832 (ETRS89 / UTM zone 32N)
+    # In PROJ >= 9.7.1 (EPSG v12.022+), it's EPSG:11022 (ETRS89-NOR / UTM zone 32N)
+    if PROJ_GTE_971:
         assert sub_crs_list[0].is_projected
         assert sub_crs_list[0] == CRS.from_epsg(11022)
     else:

